pkgs/development/node-packages/default.nix

```
@@ -35,6 +35,22 @@ let
       '';
     };
 
+    hyperspace-cli = super."@hyperspace/cli".override {
+      buildInputs = [
+        pkgs.makeWrapper
+        pkgs.nodePackages.node-gyp-build
+        pkgs.libtool
+        pkgs.autoconf
+        pkgs.automake
```

unsure about how node packages work, but this would be proper for mkDerivation
```suggestion
      nativeBuildInputs = [
          pkgs.makeWrapper
          pkgs.libtool
          pkgs.autoconf
          pkgs.automake
      buildInputs = [
        pkgs.nodePackages.node-gyp-build   
```

---

Additionally, it seems that this PR doesn't cross-compile via `pkgsCross`. It's an interesting issue. It seems also not to compile for aarch64 natively.